base/sparse/higherorderfns.jl

```diff
@@ -10,7 +10,7 @@ import Base.Broadcast: _containertype, promote_containertype,
 
 using Base: front, tail, to_shape
 using ..SparseArrays: SparseVector, SparseMatrixCSC, AbstractSparseVector,
-                      AbstractSparseMatrix, AbstractSparseArray, indtype
+                      AbstractSparseMatrix, AbstractSparseArray, indtype, nnz, nzrange
```

Comment:
These are missing imports so they fail when reloading the `Sparse` module. They are exported from `Base` and therefore automatically imported the first time but not on reloads.